experimental/tools/awps-tunnel/Tunnels/Internals/JwtBuilder.cs

```diff
@@ -0,0 +1,237 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Buffers;
+using System.Buffers.Text;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Security.Cryptography;
+using System.Text;
+using System.Text.Json;
+#nullable disable
+namespace Azure.Core
+{
+    /// <summary>
+    /// Low level library for building JWT
+    /// </summary>
+    internal class JwtBuilder : IDisposable
+    {
+        // Registered claims
+        private static byte[] s_nbf = Encoding.UTF8.GetBytes("nbf");
```

Comment:
add `readonly`

Reply:
Actually copied from https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/webpubsub/Azure.Messaging.WebPubSub/src/JwtBuilder.cs